src/createActionCreator.js

```diff
@@ -0,0 +1,23 @@
+import { createAction } from 'redux-actions';
+import toCamelCase from 'camelcase';
+
+const isFunction = value => typeof value === 'function';
+const getCreatorForType = (type, creator) => {
+  if (!creator) {
+    return creator;
+  }
+  if (isFunction(creator[type])) {
+    return creator[type];
+  }
+  if (isFunction(creator[toCamelCase(type)])) {
```

Comment:
Do we really need `toCamelCase` here?

Reply:
to I need that match long custom stages like `SOME_LOMNG_STAGE` with key `someSongStage` to add `payloadCreator` or `metaCreator`

Reply:
I can also use 
```
const toCamelCase = string => string.toLowerCase().replace(/[_.\- ]+(\w|$)/g, (_, p1) => p1.toUpperCase());
```
but it feels wrong to copy that from `camelcase` package

Reply:
maybe I can just use something like:
```
const toCamelCase = string => string.toLowerCase().replace(/_+(\w|$)/g, (_, p1) => p1.toUpperCase());
```